en/editor/extension/to-panel-messages.md

```diff
@@ -0,0 +1,70 @@
+# Communicate with the panel
+
+In general, our interaction model is dominated by "extension process" and "panel" for data presentation. To the traditional Web, the "plug-in" function is the server side, and the "panel" function is the browser on the client's computer.
+
+In this case, there is usually no direct data sent to the panel, some of the majority is some state synchronization, using broadcast broadcast can be.
```

Comment:
the majority is some state synchronization, just using `broadcast` to broadcast.